Gemfile.lock

```diff
@@ -1,114 +1,155 @@
 PATH
   remote: .
   specs:
-    bliss_attachments (0.0.4)
+    bliss_attachments (0.0.5)
       paperclip
       rails
 
 GEM
   remote: https://rubygems.org/
   specs:
-    actionmailer (3.2.14)
-      actionpack (= 3.2.14)
-      mail (~> 2.5.4)
-    actionpack (3.2.14)
-      activemodel (= 3.2.14)
-      activesupport (= 3.2.14)
-      builder (~> 3.0.0)
-      erubis (~> 2.7.0)
-      journey (~> 1.0.4)
-      rack (~> 1.4.5)
-      rack-cache (~> 1.2)
-      rack-test (~> 0.6.1)
-      sprockets (~> 2.2.1)
-    activemodel (3.2.14)
-      activesupport (= 3.2.14)
-      builder (~> 3.0.0)
-    activerecord (3.2.14)
-      activemodel (= 3.2.14)
-      activesupport (= 3.2.14)
-      arel (~> 3.0.2)
-      tzinfo (~> 0.3.29)
-    activeresource (3.2.14)
-      activemodel (= 3.2.14)
-      activesupport (= 3.2.14)
-    activesupport (3.2.14)
-      i18n (~> 0.6, >= 0.6.4)
-      multi_json (~> 1.0)
-    arel (3.0.3)
-    builder (3.0.4)
-    climate_control (0.0.3)
-      activesupport (>= 3.0)
-    cocaine (0.5.8)
-      climate_control (>= 0.0.3, < 1.0)
-    diff-lcs (1.2.4)
-    erubis (2.7.0)
-    hike (1.2.3)
-    i18n (0.6.4)
-    journey (1.0.4)
-    jquery-rails (3.0.4)
-      railties (>= 3.0, < 5.0)
+    actioncable (5.2.3)
```

Comment:
Going from 3 to 5 sounds like a bit of a stretch, doesn't it?

How did you ensure that this change is working as expected?

Reply:
We used the forked gem in our `local bliss` to update `vendor/cache` and performed testing in our machine. As we found no discrepancy in our local machine, we created an MR for [bliss](https://gitlab.com/fadendaten/bliss/merge_requests/2571) and deployed the branch to `bliss-staging`. So far, we tested, we did not find any compatibility issues in the staging environment.